### PR TITLE
security policy

### PR DESCRIPTION
**Pull request by nyrahul**
* Guidelines for reporting vulnerability
* Supported versions
* Public disclosure timings/Security bulletins

The security policy is a trimmed down version of kubernetes security policy.

### CODE REVIEW

File: SECURITY.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+| Version | Supported          |
+| ------- | ------------------ |
+| >=1.0   | :white_check_mark: |
+
+## Reporting a Vulnerability
+
+### When you should?
+- You think you discovered a potential security vulnerability in Nephio.
+- You are unsure how a vulnerability affects Nephio.
+- You think you discovered a vulnerability in the dependency of Nephio. For those projects, please leverage their reporting policy.
```
[the remark this answers is not in the document]
Comment (nyrahul) [reply SECURITY.md]: fixed

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+### When you should not?
+- You need assistance in configuring Nephio for security - please discuss this is in the [slack channel](https://nephio.slack.com/archives/C05UXLPF4V6).
+- You need help applying security-related updates.
+- Your issue is not security-related.
+
+### Please use the below process to report a vulnerability to the project:
+1. Email the **Nephio security group at sig-security@lists.nephio.org**
+
+    * Please include the requested information listed below (as much as you can provide) to help us better understand the nature and scope of the possible issue:
```
[the remark this answers is not in the document]
Comment (nyrahul) [reply SECURITY.md]: fixed

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+        * Type of issue (e.g. buffer overflow, SQL injection, cross-site scripting, etc.)
+        * Full paths of the source file(s) related to the manifestation of the issue
+        * Location of the affected source code (tag/branch/commit or direct URL) 
+        * Any special configuration required to reproduce the issue
+        * Step-by-step instructions to reproduce the issue
+        * Proof-of-concept or exploit code (if possible)
+        * Impact of the issue, including how an attacker might exploit the issue
+
+    * These information will help us triage your report more quickly.
```
[the remark this answers is not in the document]
Comment (nyrahul) [reply SECURITY.md]: fixed

**Comment by liamfallon** [comment]
It looks like this PR needs to be rebased.

**Comment by nyrahul** [comment]
> It looks like this PR needs to be rebased.

done

**Comment by liamfallon** [comment]
@nyrahul  This needs to be rebased again, apologies.

**Comment by nyrahul** [comment]
> @nyrahul This needs to be rebased again, apologies.

Hey, No problem. Done rebasing. Thanks

**Pull request merged**